Dockerfile

```diff
@@ -1,25 +1,21 @@
-FROM fedora:26
+FROM redhat/ubi8
 
-RUN dnf -y groupinstall "Development tools" \
-    && dnf -y install python-devel python-tools python3-devel python3-tools sshpass which supervisor procps\
-    && dnf clean all \
-    && rm -rf /var/cache/dnf
+RUN dnf -yq install python39 make openssh-clients glibc-langpack-en &&\
+    dnf update &&\
```

Comment:
@bruno-fs Testing for Mac, this line fails -
```
#6 7.533 Updating Subscription Management repositories.
#6 7.534 Unable to read consumer identity
#6 7.537
#6 7.537 This system is not registered with an entitlement server. You can use subscription-manager to register.
```
Because Macs can't be registered and there is no `subscription-manager`.

I commented it out locally and it built cleanly for me

Reply:
Mmmm maybe we can drop this line and just trust ubi8 packages are fresh enough

Reply:
this was addressed on #2129